src/Catalog/CommonUtility.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Services.Metadata.Catalog
+{
+    public static class CommonUtility
```

Comment:
I would rather have a common namespace to define utilities, but that would be an overkill. Will rename this to `Utilities`